src/System.Security.Cryptography.X509Certificates/ref/project.json

```diff
@@ -1,5 +1,6 @@
 {
   "dependencies": {
+    "System.Collections.NonGeneric": "4.4.0-beta-24717-02",
```

Comment:
@weshaggard In the long ago we tried keeping things like this out of Shared Framework.  I assume those days are over, but wanted to check...